app/models/category.rb

```diff
@@ -0,0 +1,12 @@
+class Category < ApplicationRecord
+  has_many :record_items
+  has_many :records, through: :record_items
+  belongs_to :author, class_name: 'User'
+
+  has_one_attached :icon
+  validates :icon, presence: true
```

Comment:
- [ ] Good job for validating the icon input field. Let's now kindly validate category name input field. Remember we were told to validate all user's input. This is to ensure that anyone with bad intentions cannot compromise our app :+1: Right now I can create a category without the name.